olwebsite-app/src/OlWebsiteApp.js

```diff
@@ -19,7 +25,12 @@ export default function OlWebsiteApp() {
         <Toolbar />
         <div className="app-body">
           <Switch>
-            <Route path="/" component={LandingView} />
+            <Route path="/about-us" exact component={LandingView} />
+            <Route path="/not-found" component={NotFoundView} />
+            <Route path="/partners" component={PartnersView} />
+            <Route path="/posts" component={PostsView} />
+            <Redirect from="/" exact to="/about-us" />
```

Comment:
Is our home page now going to just be "About Us"? I thought these were going to be separate pages. @zmeriksen @ambid17 can ya'll verify?

Reply:
I think you are right on this @will-nemo I have messaged @suzyfendrick to get a clarification.

My understanding is we will have a homepage for users that are not logged in, a dashboard for users that are.

Potentially we will have a splash page for ads... example: https://www.runescape.com/splash

About Us seems to be its own page based on Figma...

Reply:
I got the consensus from @suzyfendrick . There are two different designs on figma, but we're going with the one that has the searchbox in the toolbar.

Should I make a ticket for fixing the pages? It's not too complicated and would make a good first issue.

Reply:
good call fabrice, making an extra ticket would be great